app/views/layouts/application.html.haml

```diff
@@ -66,10 +66,15 @@
               %div.govuk-grid-column-one-third
                 %h2.govuk-heading-m
                   = t('footer.for_job_seekers')
-                %ul.govuk-footer__inline-list
-                  %li.govuk-footer__inline-list-item
+                %ul.govuk-footer__inline-list.hide-bullet-points
```

Comment:
Whats the reason for these changes?

Reply:
I think for some reason the existing list was an inline list (so they would show up next to each other).